packages/addons/addon-depends/libmbedtls/package.mk

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2022-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="libmbedtls"
+PKG_VERSION="2.16.9"
+PKG_SHA256="fc17ff7d8c11d08f23ae2800a18269408ad2c24ea6bb8b9363e41a01c2425697"
+PKG_LICENSE="GPL"
+PKG_SITE="https://www.trustedfirmware.org/projects/mbed-tls/"
+PKG_URL="https://github.com/Mbed-TLS/mbedtls/archive/refs/tags/v${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="mbed TLS (formerly known as PolarSSL) is a lean open source crypto library for providing SSL and TLS support."
+PKG_BUILD_FLAGS="-sysroot"
+
+PKG_CMAKE_OPTS_TARGET="-DCMAKE_INSTALL_PREFIX=/usr"
```

Comment:
```suggestion
PKG_CMAKE_OPTS_TARGET="-DCMAKE_INSTALL_PREFIX=/usr \
                       -DUSE_SHARED_MBEDTLS_LIBRARY=On"
```

Reply:
I was not able to build without making the mbedtls shared

Reply:
Strange. I built it again from scratch and it went fine (with mbedtls static and without any CMake option to help dislocker find it). I'm building from the `libreelec-10.0` branch

Reply:
Retested https://patch-diff.githubusercontent.com/raw/LibreELEC/LibreELEC.tv/pull/6707.patch against the libreelec-10.0

```
docker@13c067f9009f:/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0$ s/build dislocker
UNPACK      dislocker
UNPACK      fuse
    APPLY PATCH (common)      packages/sysutils/fuse/patches/fuse-0001_aarch64-support.patch
patching file include/fuse_kernel.h
    APPLY PATCH (common)      packages/sysutils/fuse/patches/fuse-0002-dont-run-update-rc.d.patch
patching file util/Makefile.in
Hunk #1 succeeded at 747 (offset 13 lines).
    FIXCONFIG      /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/
BUILD      fuse (target)
    TOOLCHAIN      configure (auto-detect)
Executing (target): /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/configure --host=x86_64-libreelec-linux-gnu --build=x86_64-linux-gnu --prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --libdir=/usr/lib --libexecdir=/usr/lib --localstatedir=/var --disable-static --enable-shared MOUNT_FUSE_PATH=/usr/sbin --enable-lib --enable-util --disable-example --enable-mtab --disable-rpath --with-gnu-ld
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-libreelec-linux-gnu
checking target system type... x86_64-libreelec-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for x86_64-libreelec-linux-gnu-strip... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-strip
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking how to print strings... printf
checking for style of include used by make... GNU
checking for x86_64-libreelec-linux-gnu-gcc... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc accepts -g... yes
checking for /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc option to accept ISO C89... none needed
checking whether /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc understands -c and -o together... yes
checking dependency style of /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc... gcc3
checking for a sed that does not truncate output... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ld
checking if the linker (/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-nm
checking the name lister (/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-libreelec-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ld option to reload object files... -r
checking for x86_64-libreelec-linux-gnu-objdump... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-objdump
checking how to recognize dependent libraries... pass_all
checking for x86_64-libreelec-linux-gnu-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for x86_64-libreelec-linux-gnu-ar... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ar
checking for archiver @FILE support... @
checking for x86_64-libreelec-linux-gnu-strip... (cached) /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-strip
checking for x86_64-libreelec-linux-gnu-ranlib... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ranlib
checking command to parse /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-nm output from /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for x86_64-libreelec-linux-gnu-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking how to run the C preprocessor... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-cpp
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc supports -fno-rtti -fno-exceptions... no
checking for /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc option to produce PIC... -fPIC -DPIC
checking if /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc PIC flag -fPIC -DPIC works... yes
checking if /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc static flag -static works... yes
checking if /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc supports -c -o file.o... yes
checking if /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc supports -c -o file.o... (cached) yes
checking whether the /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc linker (/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for x86_64-libreelec-linux-gnu-gcc... (cached) /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc accepts -g... (cached) yes
checking for /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc option to accept ISO C89... (cached) none needed
checking whether /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc understands -c and -o together... (cached) yes
checking dependency style of /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc... (cached) gcc3
checking for fork... yes
checking for setxattr... yes
checking for fdatasync... yes
checking for splice... yes
checking for vmsplice... yes
checking for utimensat... yes
checking for posix_fallocate... yes
checking for struct stat.st_atim... yes
checking for struct stat.st_atimespec... no
checking for library containing dlopen... -ldl
checking for library containing clock_gettime... none required
checking for ld used by /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc... /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ld -m elf_x86_64
checking if the linker (/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-ld -m elf_x86_64) is GNU ld... yes
checking for shared library run path origin... done
checking for iconv... yes
checking for working iconv... guessing yes
checking for iconv declaration...
         extern size_t iconv (iconv_t cd, char * *inbuf, size_t *inbytesleft, char * *outbuf, size_t *outbytesleft);
configure: UDEV_RULES_PATH env var not set, using default /etc/udev/rules.d
configure: INIT_D_PATH env var not set, using default /etc/init.d
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating fuse.pc
config.status: creating Makefile
config.status: creating lib/Makefile
config.status: creating util/Makefile
config.status: creating example/Makefile
config.status: creating include/Makefile
config.status: creating doc/Makefile
config.status: creating include/config.h
config.status: executing depfiles commands
config.status: executing libtool commands
Executing (target): make
Making all in include
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make  all-am
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
Making all in lib
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
  CC       fuse.lo
  CC       fuse_kern_chan.lo
  CC       fuse_loop.lo
  CC       fuse_loop_mt.lo
  CC       fuse_lowlevel.lo
  CC       fuse_mt.lo
  CC       fuse_opt.lo
  CC       fuse_session.lo
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse_session.c: In function 'fuse_chan_recv':
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse_session.c:211:11: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
  211 |   return ((struct fuse_chan_ops_compat24 *) &ch->op)
      |          ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CC       fuse_signals.lo
  CC       buffer.lo
  CC       cuse_lowlevel.lo
  CC       helper.lo
  CC       mount.lo
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse_lowlevel.c: In function 'fuse_add_dirent':
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse_lowlevel.c:263:2: warning: 'strncpy' output truncated before terminating nul copying as many bytes from a string as its length [-Wstringop-truncation]
  263 |  strncpy(dirent->name, name, namelen);
      |  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse_lowlevel.c:253:21: note: length computed here
  253 |  unsigned namelen = strlen(name);
      |                     ^~~~~~~~~~~~
  CC       mount_util.lo
  CC       ulockmgr.lo
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c: In function 'fuse_compat_open':
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:1536:10: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
 1536 |   err = ((struct fuse_operations_compat22 *) &fs->op)->open(path,
      |         ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:1541:10: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
 1541 |   err = ((struct fuse_operations_compat2 *) &fs->op)
      |         ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c: In function 'fuse_compat_release':
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:1552:11: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
 1552 |   return ((struct fuse_operations_compat2 *) &fs->op)
      |          ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c: In function 'fuse_compat_opendir':
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:1565:10: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
 1565 |   err = ((struct fuse_operations_compat22 *) &fs->op)
      |         ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c: In function 'fuse_compat_statfs':/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:1605:10: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
 1605 |   err = ((struct fuse_operations_compat22 *) &fs->op)
      |         ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:1612:10: warning: dereferencing type-punned pointer will break strict-aliasing rules [-Wstrict-aliasing]
 1612 |   err = ((struct fuse_operations_compat1 *) &fs->op)
      |         ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c: In function 'add_name':
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:908:2: warning: 'strncpy' output truncated before terminating nul copying as many bytes from a string as its length [-Wstringop-truncation]
  908 |  strncpy(s, name, len);
      |  ^~~~~~~~~~~~~~~~~~~~~
/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/lib/fuse.c:884:15: note: length computed here  884 |  size_t len = strlen(name);
      |               ^~~~~~~~~~~~
  CC       modules/subdir.lo
  CC       modules/iconv.lo
  CCLD     libulockmgr.la
  CCLD     libfuse.la
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
Making all in util
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make  all-am
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
  CC       fusermount-fusermount.o
  CC       fusermount-mount_util.o
  CC       ulockmgr_server-ulockmgr_server.o
  CC       mount.fuse.o
  CCLD     mount.fuse
  CCLD     ulockmgr_server
  CCLD     fusermount
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
Making all in doc
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
make[1]: Nothing to be done for 'all-am'.
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
Making install in include
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/include/fuse'
 /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_compat.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_common.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_common_compat.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_lowlevel.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_lowlevel_compat.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_opt.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/cuse_lowlevel.h '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/include/fuse'
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/include'
 /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/old/fuse.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/ulockmgr.h '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/include'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
Making install in lib
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib'
 /bin/bash ../libtool   --mode=install /usr/bin/install -c   libfuse.la libulockmgr.la '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib'
libtool: install: /usr/bin/install -c .libs/libfuse.so.2.9.9 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib/libfuse.so.2.9.9
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib && { ln -s -f libfuse.so.2.9.9 libfuse.so.2 || { rm -f libfuse.so.2 && ln -s libfuse.so.2.9.9 libfuse.so.2; }; })
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib && { ln -s -f libfuse.so.2.9.9 libfuse.so || { rm -f libfuse.so && ln -s libfuse.so.2.9.9 libfuse.so; }; })
libtool: install: /usr/bin/install -c .libs/libfuse.lai /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib/libfuse.la
libtool: install: /usr/bin/install -c .libs/libulockmgr.so.1.0.1 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib/libulockmgr.so.1.0.1
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib && { ln -s -f libulockmgr.so.1.0.1 libulockmgr.so.1 || { rm -f libulockmgr.so.1 && ln -s libulockmgr.so.1.0.1 libulockmgr.so.1; }; })
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib && { ln -s -f libulockmgr.so.1.0.1 libulockmgr.so || { rm -f libulockmgr.so && ln -s libulockmgr.so.1.0.1 libulockmgr.so; }; })
libtool: install: /usr/bin/install -c .libs/libulockmgr.lai /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib/libulockmgr.la
libtool: warning: remember to run 'libtool --finish /usr/lib'
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
Making install in util
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make  install-am
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[3]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/bin'
  /bin/bash ../libtool   --mode=install /usr/bin/install -c fusermount ulockmgr_server '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/bin'
libtool: install: /usr/bin/install -c fusermount /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/bin/fusermount
libtool: install: /usr/bin/install -c ulockmgr_server /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/bin/ulockmgr_server
/usr/bin/mkdir -p /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/sbin
/usr/bin/install -c ./mount.fuse /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/sbin/mount.fuse
/usr/bin/mkdir -p /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/etc/init.d
/usr/bin/install -c /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/util/init_script /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/etc/init.d/fuse
make  install-exec-hook
make[4]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
chmod u+s /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/bin/fusermount
mknod /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/dev/fuse -m 0666 c 10 229 || true
mknod: /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/dev/fuse: Operation not permitted
make[4]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
/usr/bin/mkdir -p /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/etc/udev/rules.d
/usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/util/udev.rules /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/etc/udev/rules.d/99-fuse.rules
make[3]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
Making install in doc
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/share/man/man1'
 /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/doc/fusermount.1 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/doc/ulockmgr_server.1 '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/share/man/man1'
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/share/man/man8'
 /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/doc/mount.fuse.8 '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/share/man/man8'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib/pkgconfig'
 /usr/bin/install -c -m 644 fuse.pc '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/.sysroot/fuse.target/usr/lib/pkgconfig'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
Making install in include
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/include/fuse'
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/include'
 /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/old/fuse.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/ulockmgr.h '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/include'
 /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_compat.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_common.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_common_compat.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_lowlevel.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_lowlevel_compat.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/fuse_opt.h /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/include/cuse_lowlevel.h '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/include/fuse'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/include'
Making install in lib
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
make[2]: Nothing to be done for 'install-data-am'.
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib'
 /bin/bash ../libtool   --mode=install /usr/bin/install -c   libfuse.la libulockmgr.la '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib'
libtool: install: /usr/bin/install -c .libs/libfuse.so.2.9.9 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib/libfuse.so.2.9.9
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib && { ln -s -f libfuse.so.2.9.9 libfuse.so.2 || { rm -f libfuse.so.2 && ln -s libfuse.so.2.9.9 libfuse.so.2; }; })
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib && { ln -s -f libfuse.so.2.9.9 libfuse.so || { rm -f libfuse.so && ln -s libfuse.so.2.9.9 libfuse.so; }; })
libtool: install: /usr/bin/install -c .libs/libfuse.lai /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib/libfuse.la
libtool: install: /usr/bin/install -c .libs/libulockmgr.so.1.0.1 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib/libulockmgr.so.1.0.1
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib && { ln -s -f libulockmgr.so.1.0.1 libulockmgr.so.1 || { rm -f libulockmgr.so.1 && ln -s libulockmgr.so.1.0.1 libulockmgr.so.1; }; })
libtool: install: (cd /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib && { ln -s -f libulockmgr.so.1.0.1 libulockmgr.so || { rm -f libulockmgr.so && ln -s libulockmgr.so.1.0.1 libulockmgr.so; }; })
libtool: install: /usr/bin/install -c .libs/libulockmgr.lai /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib/libulockmgr.la
libtool: warning: remember to run 'libtool --finish /usr/lib'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/lib'
Making install in util
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make  install-am
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[3]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
/usr/bin/mkdir -p /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/sbin
/usr/bin/mkdir -p /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/etc/udev/rules.d
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/bin'
/usr/bin/install -c ./mount.fuse /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/sbin/mount.fuse
/usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/util/udev.rules /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/etc/udev/rules.d/99-fuse.rules
/usr/bin/mkdir -p /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/etc/init.d
  /bin/bash ../libtool   --mode=install /usr/bin/install -c fusermount ulockmgr_server '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/bin'
/usr/bin/install -c /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/util/init_script /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/etc/init.d/fuse
libtool: install: /usr/bin/install -c fusermount /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/bin/fusermount
libtool: install: /usr/bin/install -c ulockmgr_server /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/bin/ulockmgr_server
make  install-exec-hook
make[4]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
chmod u+s /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/bin/fusermount
mknod /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/dev/fuse -m 0666 c 10 229 || truemknod: /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/dev/fuse: Operation not permitted
make[4]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[3]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/util'
Making install in doc
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/share/man/man1' /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/share/man/man8' /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/doc/fusermount.1 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/doc/ulockmgr_server.1 '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/share/man/man1'
 /usr/bin/install -c -m 644 /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/doc/mount.fuse.8 '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/share/man/man8'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu/doc'
make[1]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
make[2]: Entering directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib/pkgconfig'
 /usr/bin/install -c -m 644 fuse.pc '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/install_pkg/fuse-2.9.9/usr/lib/pkgconfig'
make[2]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/fuse-2.9.9/.x86_64-libreelec-linux-gnu'
BUILD      dislocker (target)
    TOOLCHAIN      cmake (auto-detect)
Executing (target): cmake -GNinja -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_TOOLCHAIN_FILE=/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/etc/cmake-x86_64-libreelec-linux-gnu.conf -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=MinSizeRel -DCMAKE_INSTALL_PREFIX=/usr -DLIB_INSTALL_DIR=/usr/lib -DCMAKE_SKIP_RPATH=ON -DWITH_RUBY=OFF /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/dislocker-0.7.3
CMake Deprecation Warning at CMakeLists.txt:20 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The C compiler identification is GNU 10.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/toolchain/bin/x86_64-libreelec-linux-gnu-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Could NOT find POLARSSL (missing: POLARSSL_INCLUDE_DIRS POLARSSL_LIBRARIES)
CMake Warning (dev) at cmake/FindPolarSSL.cmake:68 (execute_process):
  Syntax error in cmake code at

    /var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/dislocker-0.7.3/cmake/FindPolarSSL.cmake:70

  when parsing string

    s@\s\+@ @g

  Invalid escape sequence \s

  Policy CMP0010 is not set: Bad variable reference syntax is an error.  Run
  "cmake --help-policy CMP0010" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.
Call Stack (most recent call first):
  src/CMakeLists.txt:123 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

grep: POLARSSL_INCLUDE_DIRS-NOTFOUND/polarssl/version.h: No such file or directory
PolarSSL/mbedTLS version:
CMake Error at cmake/FindPolarSSL.cmake:83 (string):
  string sub-command REPLACE requires at least four arguments.
Call Stack (most recent call first):
  src/CMakeLists.txt:123 (find_package)


CMake Error at cmake/FindPolarSSL.cmake:85 (list):
  list sub-command GET requires at least three arguments.
Call Stack (most recent call first):
  src/CMakeLists.txt:123 (find_package)


CMake Error at cmake/FindPolarSSL.cmake:86 (list):
  list sub-command GET requires at least three arguments.
Call Stack (most recent call first):
  src/CMakeLists.txt:123 (find_package)


CMake Error at cmake/FindPolarSSL.cmake:87 (list):
  list sub-command GET requires at least three arguments.
Call Stack (most recent call first):
  src/CMakeLists.txt:123 (find_package)


-- Configuring incomplete, errors occurred!
See also "/var/media/DATA/home-rudi/LibreELEC.libreelec-10.0/build.LibreELEC-Generic.x86_64-10.0-devel/build/dislocker-0.7.3/.x86_64-libreelec-linux-gnu/CMakeFiles/CMakeOutput.log".
FAILURE: s/build dislocker during configure_target (default)
*********** FAILED COMMAND ***********
cmake ${CMAKE_GENERATOR_NINJA} ${TARGET_CMAKE_OPTS} ${PKG_CMAKE_OPTS_TARGET} $(dirname ${PKG_CMAKE_SCRIPT})
**************************************
```